examples/apps/data-object-grid/package.json

```diff
@@ -56,10 +56,10 @@
 		"@fluidframework/runtime-definitions": "workspace:~",
 		"@fluidframework/runtime-utils": "workspace:~",
 		"prop-types": "^15.8.1",
-		"react": "^17.0.1",
+		"react": "^18.3.1",
 		"react-collapsible": "^2.7.0",
-		"react-dom": "^17.0.1",
-		"react-grid-layout": "^0.18.3",
+		"react-dom": "^18.3.1",
+		"react-grid-layout": "^1.4.4",
```

Comment:
Went through the [changelog](https://github.com/react-grid-layout/react-grid-layout/blob/master/CHANGELOG.md) and the breaking changes don't seem to affect us.